src/index.html

```diff
@@ -21,7 +21,335 @@
     />
   </head>
 
-  <body>
-    <h1>Catalog</h1>
+  <body class="catalog">
+    <header class="catalog__header">
+      <div class="catalog__nav-logo">
+        <a href="#homepage">
+          <img
+            src="images/logo.png"
+            alt="MOYO-logo"
+          />
+        </a>
+      </div>
+
+      <nav class="catalog__nav">
+        <ul class="catalog__nav-list">
+          <li class="catalog__nav-item">
+            <a
+              class="catalog__nav-link catalog__nav-link--is-active"
+              href="#apple"
+            >
+              apple
+            </a>
+          </li>
+
+          <li class="catalog__nav-item">
+            <a
+              class="catalog__nav-link"
+              href="#samsung"
+            >
+              samsung
+            </a>
+          </li>
+
+          <li class="catalog__nav-item">
+            <a
+              class="catalog__nav-link"
+              href="#smartphones"
+            >
+              smartphones
+            </a>
+          </li>
+
+          <li class="catalog__nav-item">
+            <a
+              class="catalog__nav-link"
+              href="#laptops & computers"
+              data-qa="nav-hover"
+            >
+              laptops & computers
+            </a>
+          </li>
+
+          <li class="catalog__nav-item">
+            <a
+              class="catalog__nav-link"
+              href="#gadgets"
+            >
+              gadgets
+            </a>
+          </li>
+
+          <li class="catalog__nav-item">
+            <a
+              class="catalog__nav-link"
+              href="#tablets"
+            >
+              tablets
+            </a>
+          </li>
+
+          <li class="catalog__nav-item">
+            <a
+              class="catalog__nav-link"
+              href="#photo"
+            >
+              photo
+            </a>
+          </li>
+
+          <li class="catalog__nav-item">
+            <a
+              class="catalog__nav-link"
+              href="#video"
+            >
+              video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="catalog__product-card">
+      <section
+        class="catalog__element"
```

Comment:
Each BEM block or element that is a component should be in its own file. If `catalog__element` is a reusable component, it should be separated into its own HTML file.